src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [{ ...state }];
```

Comment:
```suggestion
  const resultArray = [];
  const clonedState =  { ...state }
```
start from empty array and work in loop with clonedState instead last item in result array